Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN yum update -y && yum install -y wget \
   && mvn package -Dmaven.test.skip=true \
   && mv target/perseo-core-*.war /usr/share/tomcat/webapps/perseo-core.war \
   && mkdir /var/log/perseo && chown tomcat:tomcat /var/log/perseo
+  # Force Tomcat to use ipv4 and optimize
+  && echo "JAVA_OPTS=\"-Djava.awt.headless=true -Xmx512m -XX:MaxPermSize=256m -XX:+UseConcMarkSweepGC -Djava.library.path=/usr/lib64:/usr/lib -Djava.security.egd=file:/dev/./urandom -Djava.net.preferIPv4Stack=true\"" >> /usr/share/tomcat/conf/tomcat.conf
```

Comment:
Even in the case Perseo FE try to connect using always IPv6 (we'll check... @crbrox could you have a look?) maybe Tomcat could be configured to run in dual-stack? Is it not running in dual-stack by default?